manifests/init.pp

```diff
@@ -98,8 +98,34 @@
   $epel_testing_debuginfo_includepkgs     = undef,
   $epel_testing_debuginfo_sslclientkey    = undef,
   $epel_testing_debuginfo_sslclientcert   = undef,
-  Boolean $epel_gpg_managed               = true,
-  $os_maj_release                         = $epel::params::os_maj_release,
+  String $epel_playground_mirrorlist      = "https://mirrors.fedoraproject.org/${url_path}?repo=playground-epel${os_maj_release}&arch=\$basearch",
+  String $epel_playground_baseurl         = 'absent',
```

Comment:
can you please check for which of those empty strings are actually valild? Otherwise please update to `String[1]` to prohibit empty strings

Reply:
not sure I follow what you mean here. baseurl definition is here https://puppet.com/docs/puppet/5.5/types/yumrepo.html#yumrepo-attribute-baseurl same a mirror list it accept `absent` and any string

Reply:
the type accepts any thing, but does it make sense to pass an empty string like `''`? I don't think so. In that case`String[1]` would make sense to prohibit empty strings.